### PR TITLE
Additional documentation on AppConfig

### PR DESCRIPTION
**Pull request by techdragon**
Some advice to help avoid users breaking their basic django permissions

### CODE REVIEW

**Comment by auvipy** [comment]
ci build is failing

**Comment by techdragon** [comment]
@auvipy Im fairly sure that might not be the fault of my PR fault. :wink: I'm struggling to find a way that changing one of the restructured text doc files might cause the failures I see in the Travis CI results :smile:

**Comment by justquick** [comment]
yea that has nothing to do w/ the docs. if adding more docs break the ci builds im doing something terribly wrong

File: docs/source/configuration.rst [hunk 1]
```diff
@@ -43,6 +43,33 @@ For Django versions 1.7 or later, you should use `AppConfig <https://docs.django
 
     Introducing the registry change makes the ``ACTSTREAM_SETTINGS['MODELS']`` setting obsolete so please use the register functions instead.
 
+
+Registering Django's default User model from ``contrib.auth.user`` requires a more sophisticated AppConfig class.
+    
+.. code-block:: python
```
Comment (justquick) [inline docs/source/configuration.rst]: where does this file belong? actstream/apps.py or myapp/apps.py

**Comment by justquick** [comment]
@techdragon 
what does this changeset solve? im not sure i understand the problem why this AppConfig is required in the first place. you mention something about permissions, can you be more specific?

**Comment by techdragon** [comment]
@justquick - Its just some additional information for users of the library to help avoid making a possible mistake. Since its an easy mistake to make while doing something many users will do with this library, a and it has subtle consequences that may not be noticed for quite some time the warning seemed worth providing.

**Comment by justquick** [comment]
you are going to have to be waaay more specific about the problem here

**Comment by justquick** [comment]
@techdragon im not sure what problem you are trying to solve. can you post the issue you are seeing instead of the code that fixes it? ur solution might not be the best way

**Pull request closed without merging**